src/services/rfqm_service.ts

```diff
@@ -171,6 +171,25 @@ export class RfqmService {
         return quotedMakerAmount.div(quotedTakerAmount).times(sellAmount).decimalPlaces(0);
     }
 
+    private static _validateJob(job: RfqmJobEntity): string | undefined {
```

Comment:
consider wrapping the string in an `Error`

Reply:
agreed 👍